source-code/website/src/pages/editor/@host/@owner/@repository/State.tsx

```diff
@@ -455,8 +478,7 @@ export function EditorStateProvider(props: { children: JSXElement }) {
 		if (!inlangConfig.error && inlangConfig() && fsChange()) {
 			// setting the origin store because this should not trigger
```

Comment:
We have a blocker.
We set the resources with `setOriginResources` in line 455, then write everything to disk, with `writeResources` which at the end updates `fsChange`. This triggers this code and calls `readResources`. But read resources will only read the old files from the file system. Then newly written resource does not get loaded.

Reply:
fixed it!